formatTest/typeCheckedTests/expected_output/mlSyntax.re

```diff
@@ -101,8 +101,6 @@ module EM = {
 
 exception Ealias = EM.E;
 
-let switch_ = "switch";
+let match = "match";
```

Comment:
Yeah, basically:

.ml -> .re
switch -> match
pub -> method
pri -> private

and vice versa.